Source/Libraries/Adapters/MySqlAdapters/MySqlOutputAdapter.cs

```diff
@@ -176,17 +176,12 @@ protected override void ProcessMeasurements(IMeasurement[] measurements)
                 foreach (IMeasurement measurement in measurements)
                 {
                     // Create the command string to insert the measurement as a record in the table.
-                    StringBuilder commandString = new StringBuilder("INSERT INTO Measurement VALUES ('");
                     IDbCommand command = m_connection.CreateCommand();
+                    command.Parameters.Add(measurement.ID);
+                    command.Parameters.Add((long)measurement.Timestamp);
+                    command.Parameters.Add(measurement.AdjustedValue);
 
-                    commandString.Append(measurement.ID);
-                    commandString.Append("','");
-                    commandString.Append((long)measurement.Timestamp);
-                    commandString.Append("',");
-                    commandString.Append(measurement.AdjustedValue);
-                    commandString.Append(')');
-
-                    command.CommandText = commandString.ToString();
+                    command.CommandText = "INSERT INTO Measurement VALUES ({0}, {1}, {2})";
```

Comment:
I missed this. Format strings only work with `AdoDataConnection`. Otherwise, you need to use the SQL syntax for parameters. In MySQL, that'd be `@parameterName`.